### PR TITLE
adds vitals form

### PR DESCRIPTION
**Pull request by semujju**
https://app.asana.com/0/1133167201254915/1134274907868858

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/vitals.xml [hunk 1]
```diff
@@ -0,0 +1,256 @@
+<htmlform formUuid="a000cb34-9ec1-4344-a1c8-f692232f6edd"
+          formName="Vitals"
+          formEncounterType="67a71486-1a54-468f-ac3e-7091a9a79584"
+          formVersion="1.0"
+          formDescription="Client HIV Testing Card"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/vitals.xml]: @semujju form name - Really?

Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: @ssmusoke i didn't have the right name at the time, used that as a place holder.

Comment (slubwama) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: @ssmusoke. Did you mean Name or Description. I find the name fine. @semujju Kindly Work on the Form Description

**Comment by semujju** [comment]
I have created the vitals form and set it to display with the simpleUI by default (formDisplayStyle="Simple"), this work wells as shown in the screen shot below.
Screenshot from 2019-08-21 17-20-01

I need guidance on how to set the form to display with the simpleUI on editing form data. When editing vitals, the form shows up with a standardUI by default as seen in the screen shot below.

Screenshot from 2019-08-21 17-25-51

File: omod/src/main/webapp/resources/htmlforms/vitals.xml [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+
+    .encounter-summary-container #calculated-bmi {
+    font-size: 1em;
+    font-weight: normal;
+    }
+    #99030{
+    padding: 5px;
+    }
+
+    .htmlformform form input, form select, form textarea, form ul.select, .form input, .form select, .form textarea, .form ul.select {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/vitals.xml]: @semujju Move these form styles into the global stylesheet

File: omod/src/main/webapp/resources/htmlforms/vitals.xml [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+      formVersion="1.0"
+      formDescription="Vitals Form"
+      formAddMetadata="yes"
+      formUILocation="patientDashboard.visitActions"
+      formOrder="8"
+      formIcon="icon-medkit"
+      formShowIf="(visit.active || !visit.active)"
+      formDisplayStyle="Simple"
+      formLabel="Vitals Form">
+<style>
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/vitals.xml]: @semujju Add indentation to all Javascript and CSS styles

File: omod/src/main/webapp/resources/htmlforms/vitals.xml [hunk 4]
```diff
@@ -0,0 +1,329 @@
+<htmlform formUuid="a000cb34-9ec1-4344-a1c8-f692232f6edd"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/vitals.xml]: @semujju rename file to Vitals.xml

File: omod/src/main/webapp/resources/htmlforms/vitals.xml [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+            jq('#bp_systolic input[type=text]').attr('min', '50');
+            });
+
+            jq(function() {
+            jq('#bp_diastolic input[type=text]').attr('min', '30');
+            });
+
+        </script>
+    </ifMode>
+
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/vitals.xml]: @semujju Add code to redirect to simple UI 

<ifMode mode="EDIT">
        <script type="text/javascript">
            if (window.location.href.indexOf('editHtmlFormWithStandardUi') !== -1) {
                window.location.href = window.location.href.replace('editHtmlFormWithStandardUi','editHtmlFormWithSimpleUi');
            }
        </script>
    </ifMode>

**Comment by ssmusoke** [comment]
@semujju add right margin of 5px between the text field and text 

Screenshot 2019-09-03 12 20 51

File: omod/src/main/webapp/resources/htmlforms/vitals.xml (hunk 2 above)
Comment (chosendev1) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: done

File: omod/src/main/webapp/resources/htmlforms/vitals.xml (hunk 3 above)
Comment (chosendev1) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: done

File: omod/src/main/webapp/resources/htmlforms/vitals.xml (hunk 2 above)
Comment (chosendev1) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: Done

Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/vitals.xml (hunk 3 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/vitals.xml (hunk 4 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/vitals.xml [hunk 6]
```diff
@@ -0,0 +1,329 @@
+<htmlform formUuid="a000cb34-9ec1-4344-a1c8-f692232f6edd"
+      formName="Vitals"
+      formEncounterType="68a71486-1a54-468f-ac3e-7091a9a79584"
+      formVersion="1.0"
+      formDescription="Vitals Form"
+      formAddMetadata="yes"
+      formUILocation="patientDashboard.visitActions"
+      formOrder="8"
+      formIcon="icon-medkit"
+      formShowIf="(visit.active || !visit.active)"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/vitals.xml]: @semujju Add a check for a person who is not dead

File: omod/src/main/webapp/resources/htmlforms/vitals.xml [hunk 7]
```diff
@@ -0,0 +1,329 @@
+<htmlform formUuid="a000cb34-9ec1-4344-a1c8-f692232f6edd"
+      formName="Vitals"
+      formEncounterType="68a71486-1a54-468f-ac3e-7091a9a79584"
+      formVersion="1.0"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/vitals.xml]: @semujju Change version to 3.0

Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: done

File: omod/src/main/webapp/resources/htmlforms/vitals.xml (hunk 6 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/vitals.xml]: Done

**Review by ssmusoke — APPROVED** [review]

**Pull request merged**